css/overlay.scss

```diff
@@ -9,53 +9,3 @@
 	}
```

Comment:
these rules need adjusting or can be deleted?
At least the classes dont match anymore or is the vue lib using the same lib below?